js/client-mainmenu.js

```diff
@@ -1158,13 +1206,15 @@
 				this.$el.html('<ul class="popupmenu">' + bufs[0] + '</ul>');
 			}
 		},
-		moreTeams: function () {
+		moreTeamsPress: function () {
```

Comment:
Not sure about this rename - I don't think we need action verbs (press, click, toggle, etc) in the names?

Reply:
Changed the name because this.moreTeams was erroring, as there was the this.moreTeam instance and the this.moreTeams method, and it was picking up the method. Not too familiar with backbone js, so if there's an alternative let me know

Reply:
Notice that I also created the instance field moreTeams, so I could manipulate it in the other action methods (before it was literally just false -> true, with no toggle. this also makes it a lot easier to toggle other teams, if needed)

Reply:
Ah yes, I missed that you added a new instance field, which would necessitate renaming this.